src/common/cid.ts

```diff
@@ -3,6 +3,9 @@ import { decode as decodeMultihash } from "multiformats/hashes/digest"
 import { hasProp, isNum, isObject } from "./type-checks.js"
 
 
+export { CID }
```

Comment:
So people don't necessarily need to add the multiformats library as a dependency? Because most people don't know what that is anyway.